app_dev.yaml

```diff
@@ -142,3 +142,4 @@ env_variables:
   DATASTORE_HOST: "http://localhost:8089"
   DATASTORE_PROJECT_ID: "dev-project-id"
   DATASTORE_USE_PROJECT_ID_AS_APP_ID: "true"
+  SECRETS: "{\"ANDROID_BUILD_SECRET\": \"android-dev-secret\"}"
```

Comment:
Is it possible to make the name and the (I assume) default value match?

Reply:
@BenHenning Is this actually needed anymore?

Reply:
Yep, we can't use the scripts argument because it doesn't work correctly (and causes run failures in the dev server). We can change this value to anything so long as it's kept relatively consistent (so that we can update the corresponding downloads script).

Reply:
Okay, changed to `android-build-secre`